torch/_inductor/codegen/wrapper.py

```diff
@@ -1165,6 +1174,17 @@ def traverse(cur_kernel):
                         compile_wrapper.newline()
                         compile_wrapper.writeline(f"{symbol_name} = {symbol!r}")
                         symbols_included.add(symbol_name)
+                    elif (
+                        symbol_name in unqualified_loads
+                        and symbol_name != "tl"  # already imported
+                        and hasattr(symbol, "__module__")
+                    ):
+                        # a global symbol referenced without module
+                        # qualification: need to codegen an import
+                        compile_wrapper.writeline(
```

Comment:
Hmm perhaps we should have a "safe" list of things we can emit here? otherwise we will end up executing import time deps of unknown code.

wdyt?

Reply:
We can limit to the module names starting from `triton.` maybe? As other things are not usable in Triton kernels anyway. Do you think that would be safe enough? One thing is that, if something unqualified is *used* in the kernel code and exists in globals and has a `__module__`, we likely should import it for the kernel to be compileable.